README.md

```diff
@@ -30,45 +35,77 @@ When the reader has completed this Code Pattern, they will understand how to:
 * [Data Science](https://medium.com/ibm-watson): Systems and scientific methods to analyze structured and unstructured data in order to extract knowledge and insights.
 * [Python](https://www.python.org/): Python is a programming language that lets you work more quickly and integrate your systems more effectively.
 
-# Watch the Video
-
 # Steps
 
-## Run locally
+Either Run locally:
 
 1. [Clone the repo](#1-clone-the-repo)
-1. [Run the notebook](#2-run-the-notebook)
+2. [Run the notebook](#2-run-the-notebook)
```

Comment:
but then you have "Run the notebook" again in step 5.  Maybe you meant to call this something like "Open the notebook", but seems like it might be easier if you just have a short "Run"  section both here and for studio below. Then you wouldn't need to jump to 5.   Just have it a little redundant.